test/unittests/mongoBackend/mongoContextProvidersUpdate_test.cpp

```diff
@@ -1613,33 +1540,32 @@ TEST(mongoContextProvidersUpdateRequest, severalCprs)
   ASSERT_EQ(2, RES_CER(2).contextAttributeVector.size());
 
   EXPECT_EQ("A5", RES_CER_ATTR(2, 0)->name);
-  EXPECT_EQ("T", RES_CER_ATTR(2, 0)->type);
+  EXPECT_EQ("", RES_CER_ATTR(2, 0)->type);
```

Comment:
As in the case of queryContext not found or CPr-ed attributes, the type is set to "" (I think it should be a problem: when serviceRouting forward the update it will use again the type in the request, not the one that mongoBackend is setting in this place).